integrations/langchain/document-loaders/spider-web-scraper-crawler.md

```diff
@@ -1,10 +1,12 @@
 ---
-description: Scrape & Crawl the web with Spider.
+description: Scrape & Crawl the web with Spider - the fastest open source web scraper & crawler.
 ---
 
 # Spider Web Scraper/Crawler
 
-<figure><img src="../../../.gitbook/assets/up-017.png" alt="" width="365"><figcaption><p>Spider Web Scraper/Crawler Node</p></figcaption></figure>
+<figure><img src="../../../.gitbook/assets/spider.png" alt="" width="365"><figcaption><p>Spider Web Scraper/Crawler Node</p></figcaption></figure>
```

Comment:
do you want to upgrade the docs to a step by step guide instead of having the section is a work in progress?